### ISSUE TITLE
URLs(?) count differently against character limit in Tusky and web

### ISSUE THREAD

**Issue by mcclure** [tuskyapp/Tusky#3849]
Consider this exact post

```
What I'm listening to today: "four ( empress effects zoia, moog mother 32, make noise 0-coast )", i saw the SINE wav

Some excellent ambient/drone. Seems at first like it's just going to be unattended wom wom wom zone-out noises for 13 minutes, which would also have been good, but a few minutes in the musician starts more actively guiding things and some DC-offset popping noises that appear at first to be recording errors start to clump together into rhythmic percussion.

https://www.youtube.com/watch?v=7EPYU-1q_fw
```

This is at -4 against the character limit in Tusky and 0 on mastodon.social's web interface. I don't know why.

![image](https://github.com/tuskyapp/Tusky/assets/277318/a9033294-e2a2-44c4-a406-c1f1c940b8ba)

* * * *
- Tusky Version: 21.0 with trivial patches
- Android Version: 12
- Android Device: XPeria 5 III
- Mastodon instance (if applicable): mastodon dot social

- [X] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by nikclayton** [comment]
This was fixed in 22.0

**Comment by mcclure** [comment]
Whoops
